torch/fx/experimental/symbolic_shapes.py

```diff
@@ -4653,7 +4658,7 @@ def trivial_solve(lhs, rhs):
                             # Propagate the value ranges.  It doesn't really
                             # matter if we use truediv or floordiv, because we
                             # have established divisibility.
-                            self._update_var_to_range(i1, SymPyValueRangeAnalysis.truediv(
+                            self._update_var_to_range(i1, SymPyValueRangeAnalysis.floordiv(
```

Comment:
I'm not sure this actually does anything lol

Reply:
?
Don't we have tests that test these things?